example/multiple-things.js

```diff
@@ -6,7 +6,7 @@ const {
   Thing,
   Value,
   WebThingServer,
-} = require('../index');
+} = require('webthing-node');
```

Comment:
If we're doing it this way, these should just be `require('webthing')`, as `webthing` is the actual Node package name.